src/optimize/dynamic_dll_plugin/dll_compiler.js

```diff
@@ -260,17 +260,6 @@ export class DllCompiler {
                 notAllowedModules.push(module.resource);
                 return;
               }
-
-              // Even when the reason for the module comes from
-              // node_modules directory, assure it's not from
-              // node_modules/x-pack source code but from a real node_module
-              const dirs = reason.module.resource.split(path.sep);
```

Comment:
@mistic I don't believe this is necessary anymore as X-Pack is no longer a node module itself. Can you confirm?

Reply:
@tylersmalley as you are removing x-pack from the node_modules directory this is true! We don't need that piece of code anymore